src/spikeinterface/core/job_tools.py

```diff
@@ -28,8 +27,9 @@
                     Total memory usage (e.g. "500M", "2G")
                 - chunk_duration : str or float or None
                     Chunk duration in s if float or with units if str (e.g. "1s", "500ms")
-            * n_jobs: int
-                Number of jobs to use. With -1 the number of jobs is the same as number of cores
+            * n_jobs: int | float
+                Number of jobs to use. With -1 the number of jobs is the same as number of cores.
+                Using a float between 0 and 1 will use that fraction of the total cores.
```

Comment:
I'm wondering if we should also make clear that technically someone can use n_jobs > number of cores which can actually degrade performance by splitting off too many processes? I didn't actually realize this until I looked at #2817.

Reply:
Can you open an issue about this?

Reply:
Done!